src/components/pagination/__tests__/__snapshots__/pagination.test.js.snap

```diff
@@ -65,7 +66,7 @@ exports[`Pagination Component should render a non-connected component: non-conne
       "items": "",
       "itemsPerPage": "Items per page",
       "ofWord": "of",
-      "optionsToggle": "Items per page",
+      "optionsToggle": "",
```

Comment:
optional/future:
we may have to set this prop manually now, confusing why PF decided to not continue pulling placeholder like content for this. keep an eye on it